cosmos-tidoop-api/src/cmd_runner.js

```diff
@@ -38,7 +38,16 @@ function runHadoopJar(userId, jarName, jarInHDFS, className, libJarsName, libJar
 
         command.on('close', function(code) {
             // Run the MR job
-            var params = ['-u', userId, 'hadoop', 'jar', '/home/' + userId + '/' + jarName, className, '-libjars', '/home/' + userId + '/' + libJarsName, input, output];
+            //var params = ['-u', userId, 'hadoop', 'jar', '/home/' + userId + '/' + jarName, className, '-libjars', '/home/' + userId + '/' + libJarsName, input, output, otherArgs];
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/pull/183/commits/2cd16c45b70422b77ed2cec1720a89e23630f3c2